lib/Controller/DisplayGroup.php

```diff
@@ -582,10 +582,11 @@ public function add()
     public function edit($displayGroupId)
     {
         $displayGroup = $this->displayGroupFactory->getById($displayGroupId);
-        $preEditIsDynamic = $displayGroup->isDynamic;
+        $preEditIsDynamic = $displayGroup->getOriginalValue('isDynamic');
```

Comment:
haha, i meant you didn't have to set it up here and that you could just use the getOriginalValue below (i.e. you don't need to store it)

NM though :-)